hw3/01-characters.js

```diff
@@ -0,0 +1,33 @@
+const character = document.getElementsByClassName("character");
+
+async function getCharacters() {
+  const url = "https://thronesapi.com/api/v2/Characters";
+  try {
+    let response = await fetch(url);
+    return await response.json();
+  } catch (erorr) {
+    console.log(error);
+  }
+}
```

Comment:
I think this is a typo which can be fixed(erorr -> error) and maybe adding a error message will improve the readability.